src/components/Login.js

```diff
@@ -9,24 +10,24 @@ const Login = ({ login, status }) => {
             <span>Trying to restore authentication...</span>
           )}
           {status === 'restored' && (
-            <div>
-              <img
+            <LoginBox>
+              <Logo
                 src="https://res.cloudinary.com/teapig/image/upload/v1595701200/wevote/logo_blanco.png"
                 alt="logo"
               />
-              <button type="button" onClick={login}>
+              <LoginButton type="button" onClick={login}>
```

Comment:
Queda por defecto en type="button" cuando usás styled-components
```suggestion
              <LoginButton onClick={login}>
```